src/miniscript/mod.rs

```diff
@@ -459,6 +458,14 @@ impl_from_str!(
 
 serde_string_impl_pk!(Miniscript, "a miniscript", Ctx; ScriptContext);
 
+/// Provides a Double SHA256 `Hash` type that displays forwards.
+pub mod hash256 {
+    use bitcoin::hashes::{hash_newtype, sha256d};
+
+    #[rustfmt::skip]
+    hash_newtype!(Hash, sha256d::Hash, 32, doc = "A bitcoin block hash.", false);
```

Comment:
Merging this now to fix the CI. But we should update the doc-comment to reflect that this is just double-hashed sha256::Hash with display forward and not a bitcoin block hash.